lib/web/static/js/app.js

```diff
@@ -0,0 +1,14 @@
+const ResourceList = require('./resource_list.js');
+const listHolders = $('.holder');
+const lists = {}
+
+$.each(listHolders, function(_i, holder) {
+  let list = new ResourceList(holder)
+  let state = holder.dataset.state
+  lists[state] = list
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)